From 29d0b1d26ce9214bfcfdb5383d9f3ab9b9ec6105 Mon Sep 17 00:00:00 2001
From: Manjukumar Matha <manjukumar.harthikote-matha@xilinx.com>
Date: Sat, 26 Nov 2016 21:27:39 -0800
Subject: [PATCH] packagegroup-petalinux.bb: Add libmali-xlnx

Signed-off-by: Manjukumar Matha <manjukumar.harthikote-matha@xilinx.com>
---
 recipes-core/packagegroups/packagegroup-petalinux.bb | 1 +
 1 file changed, 1 insertion(+)

diff --git a/recipes-core/packagegroups/packagegroup-petalinux.bb b/recipes-core/packagegroups/packagegroup-petalinux.bb
index 2aa91bf..90f3516 100644
--- a/recipes-core/packagegroups/packagegroup-petalinux.bb
+++ b/recipes-core/packagegroups/packagegroup-petalinux.bb
@@ -181,6 +181,7 @@ RDEPENDS_${PN}_append_zynq += " \
 	"
 
 RDEPENDS_${PN}_append_zynqmp += " \
+	libmali-xlnx \
 	${ZYNQ_EXTRAS} \
 	${QT_EXTRAS} \
 	${@bb.utils.contains('DISTRO_FEATURES', 'x11', ' xserver-xorg-extension-glx xf86-video-armsoc ${XSERVER} \
-- 
2.1.4

